src/cpp/rtps/transport/TCPTransportInterface.cpp

```diff
@@ -1035,14 +1039,18 @@ bool TCPTransportInterface::CreateInitialConnect(
 
     std::vector<fastdds::rtps::IPFinder::info_IP> local_interfaces;
     // Check if the locator is from an owned interface to link all local interfaces to the channel
-    is_own_interface(physical_locator, local_interfaces);
-    if (!local_interfaces.empty())
+    // Note: Only appliable for TCPv4 until TCPv6 scope selection is implemented
```

Comment:
NIT, just more common to say
```suggestion
    // Note: Only applicable to TCPv4 until TCPv6 scope selection is implemented
```